Src/PerformanceCollector/Shared/Implementation/QuickPulse/Service contract/TelemetryDocument/EventTelemetryDocument.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Microsoft.ManagementServices.RealTimeDataProcessing.QuickPulseService
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    internal struct EventTelemetryDocument : ITelemetryDocument
+    {
+        [DataMember(EmitDefaultValue = false)]
+        public Guid Id { get; set; }
```

Comment:
what does this GUID represents? Please do not diverge from schema too much. Remember, you promised to start using built-in types